addon/components/cell.js

```diff
@@ -52,26 +68,45 @@ export default Component.extend(PropTypeMixin, {
 
   getDefaultProps () {
     return {
-      readOnly: false
+      readOnly: false,
+      propagatedValue: {},
+      propagatedValueChangeSet: null
+    }
+  },
+
+  didReceiveAttrs ({oldAttrs, newAttrs}) {
+    const valueChangeSet = this.get('valueChangeSet')
+    const oldCellConfig = _.get(oldAttrs, 'cellConfig.value')
+    const cellConfig = this.get('cellConfig')
```

Comment:
I personally like to use the prefix `new` when in `didReceiveAttrs`, in this case I'd use `newCellConfig`.